### DISCUSSION TITLE
Get the username of the last created user

### DISCUSSION THREAD

**Discussion by MuslemRahimi** [pocketbase/pocketbase#2225] [Q&A] [answered]
Hi, 

the title summarizes the question well. I want to print out the last created user in my database in the form of

`const output = await locals.pb.collection('users').last`

**Comment by uteminhduc** [comment, marked as answer]
Sort by `-created` and get one page, one item

```javascript
const resultList = await pb.collection('users').getList(1, 1, {
    sort: '-created',
});
```

**Reply by MuslemRahimi** [reply]
thank you so much
